gallery/src/styles/NxReadOnly/NxReadOnlyDlExample.html

```diff
@@ -0,0 +1,29 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+<dl class="nx-read-only">
+  <dt class="nx-read-only__label">
```

Comment:
This example only has one `dt`.  I think it needs several `dt`/`dd` groups to show the spacing between them.

Reply:
> This example only has one `dt`. I think it needs several `dt`/`dd` groups to show the spacing between them.

I flashed out the examples here: https://github.com/sonatype/sonatype-react-shared-components/pull/243/commits/f03dab91c22292c7ee5753fca1e53ad3238c23bc